modules/sekindoUMBidAdapter.js

```diff
@@ -1,91 +1,113 @@
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function(validBidRequests, bidderRequest) {
     var pubUrl = null;
-    if (parent !== window) { pubUrl = document.referrer; } else { pubUrl = window.location.href; }
-
-    for (var i = 0; i < bidsCount; i++) {
-      var bidReqeust = bids[i];
-      var callbackId = bidReqeust.bidId;
-      _requestBids(bidReqeust, callbackId, pubUrl);
-      // store a reference to the bidRequest from the callback id
-      // bidmanager.pbCallbackMap[callbackId] = bidReqeust;
+    if (parent !== window) {
+      pubUrl = document.referrer;
+    } else {
+      pubUrl = window.location.href;
     }
-  }
-
-  $$PREBID_GLOBAL$$.sekindoCB = function(callbackId, response) {
-    var bidObj = getBidRequest(callbackId);
-    if (typeof (response) !== 'undefined' && typeof (response.cpm) !== 'undefined') {
-      var bid = [];
-      if (bidObj) {
-        var bidCode = bidObj.bidder;
-        var placementCode = bidObj.placementCode;
 
-        if (response.cpm !== undefined && response.cpm > 0) {
-          bid = bidfactory.createBid(CONSTANTS.STATUS.GOOD);
-          bid.callback_uid = callbackId;
-          bid.bidderCode = bidCode;
-          bid.creative_id = response.adId;
-          bid.cpm = parseFloat(response.cpm);
-          bid.ad = response.ad;
-          bid.width = response.width;
-          bid.height = response.height;
+    return validBidRequests.map(bidRequest => {
+      var subId = utils.getBidIdParameter('subId', bidRequest.params);
+      var spaceId = utils.getBidIdParameter('spaceId', bidRequest.params);
+      var bidfloor = utils.getBidIdParameter('bidfloor', bidRequest.params);
+      var protocol = (document.location.protocol === 'https:' ? 's' : '');
+      var queryString = '';
 
-          bidmanager.addBidResponse(placementCode, bid);
-        } else {
-          bid = bidfactory.createBid(CONSTANTS.STATUS.NO_BID);
-          bid.callback_uid = callbackId;
-          bid.bidderCode = bidCode;
-          bidmanager.addBidResponse(placementCode, bid);
+      queryString = utils.tryAppendQueryString(queryString, 's', spaceId);
+      queryString = utils.tryAppendQueryString(queryString, 'subId', subId);
+      queryString = utils.tryAppendQueryString(queryString, 'pubUrl', pubUrl);
+      queryString = utils.tryAppendQueryString(queryString, 'hbTId', bidRequest.transactionId);
+      queryString = utils.tryAppendQueryString(queryString, 'hbBidId', bidRequest.bidId);
+      queryString = utils.tryAppendQueryString(queryString, 'hbver', '4');
+      queryString = utils.tryAppendQueryString(queryString, 'hbcb', '1');/// legasy
+      queryString = utils.tryAppendQueryString(queryString, 'dcpmflr', bidfloor);
+      queryString = utils.tryAppendQueryString(queryString, 'protocol', protocol);
+      if (bidRequest.mediaType === 'video') {
```

Comment:
May want to check for `mediaTypes.video` as well, see other comment for related info

Reply:
done